src/helper/formValidate.js

```diff
@@ -0,0 +1,38 @@
+export const titleValidate = (form, name) => {
+  const input = form.querySelector(`input[name=${name}]`);
+  input.addEventListener('invalid', (e) => {
+    if (e.target.validity.valueMissing) {
+      e.target.setCustomValidity('Title cannot be empty!');
+    }
+  });
+
+  input.addEventListener('change', (e) => {
+    e.target.setCustomValidity('');
+  });
+};
+
+export const authorValidate = (form, name) => {
+  const input = form.querySelector(`input[name=${name}]`);
+  input.addEventListener('invalid', (e) => {
+    if (e.target.validity.valueMissing) {
+      e.target.setCustomValidity('Author cannot be empty!');
+    }
+  });
+
+  input.addEventListener('change', (e) => {
+    e.target.setCustomValidity('');
+  });
+};
+
+export const numberValidate = (form, name) => {
```

Comment:
These three functions are quite similar. can you combine 3 functions into 1?

Reply:
ok I'll give it a try.